src/qt/bitcoingui.cpp

```diff
@@ -1151,7 +1151,7 @@ void BitcoinGUI::setNumConnections(int count)
     default: icon = ":/icons/connect_4"; break;
     }
     labelConnectionsIcon->setPixmap(QIcon(icon).pixmap(STATUSBAR_ICONSIZE,STATUSBAR_ICONSIZE));
-    labelConnectionsIcon->setToolTip(tr("%n active connection(s) to Gridcoin network", "", count));
+    labelConnectionsIcon->setToolTip(tr("%1 active connection(s) to Gridcoin network").arg(count));
```

Comment:
Sorry it was a dumb question, get your point now^^ It is early in the morning. If I'd write a script to change all the translation strings would that be okay than?